### ISSUE TITLE
Feature points gone wrong

### ISSUE THREAD

**Issue by qwemaze** [gordalina/faced#7]
[example](http://i.imgur.com/ziM7kLw.png)
``` javascript
let draw = (feature, color) => {
        console.log("Feature",
            feature.getX(),
            feature.getX2(),
            feature.getY(),
            feature.getY2(),
            feature.getWidth(),
            feature.getHeight()
        );
        image.rectangle(
            [feature.getX(), feature.getY()],
            [feature.getX2(), feature.getY2()],
            color,
            2
        );
    }
```

```
Feature 54 157 57 160 103 103
Feature 78 132 128 160 54 32
Feature 80 121 110 144 41 34
Feature 66 117 77 111 51 34
Feature 102 152 76 110 50 34
```

**Comment by PragyaTripathi** [comment]
The reason is the rectangle function. According to http://docs.opencv.org/2.4/modules/core/doc/drawing_functions.html (it doesn't have a js example), you should pass [x,y], [width,height] Not [x,y], [width+x, height+y]. I confirmed this further by looking into their C++ code. OpenCV seems to take care of it for you. ```
cv::rectangle(self->mat, cv::Point(x, y), cv::Point(x + width, y + height),
        color, thickness);
```
getX2 and getY2 should be replaced with getWidth and getHeight subsequently. Let me submit a pull request.

**Comment by gordalina** [comment]
Released `1.1.4`
